src/app/store/Features/OldboyBarbershop/oldboySlice.ts

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { createSlice } from '@reduxjs/toolkit';
+
+export interface OldboyState {
+  value: string;
+}
+
+const initialState: OldboyState = {
+  value: 'Hello world!',
+};
+
+export const OldBoySlice = createSlice({
+  name: 'oldboyBarbershop',
```

Comment:
`oldBoyBarbershop` - we will match other places where you typed `B` with upper case letters